operator-common/src/main/java/io/strimzi/operator/common/model/cruisecontrol/CruiseControlParameters.java

```diff
@@ -85,7 +85,22 @@ public enum CruiseControlParameters {
     /**
      * Broker ID
      */
-    BROKER_ID("brokerid");
+    BROKER_ID("brokerid"),
+
+    /**
+     * Topic
+     */
```

Comment:
Is this really needed? But as I see you just follow the pattern of the whole class...